Gemfile

```
@@ -5,26 +5,21 @@ gem 'hiera-eyaml',                                               '~> 3.2', '>= 3
 gem 'json',                                                      '~> 2.3'
 gem 'nokogiri',                                                  '~> 1.10', '>= 1.10.9'
 gem 'onceover',                                                  '~> 3.20', '>= 3.20.0'
-gem 'onceover-codequality',                                      '~> 0.8', '>= 0.8.0'
+gem 'onceover-codequality',                                      '~> 0.9', '>= 0.9.0'
```

Those version restraints don't make any sense? Isn't it enough to pin to:
```suggestion
gem 'onceover-codequality',                                      '~> 0.9'
```

---

can do, only edited it to the version i wanted, didn't think about the syntax 😄